### PR TITLE
Shallow Contextual Biasing for Whisper

### PR DESCRIPTION
**Pull request by zwycl**
Adds an optional contextual biasing parameter to the Whisper model to enable shallow contextual biasing toward given sequences by modifying logits. This is a flexible and fairly simple method useful for transcribing out-of-vocabulary entities in ASR or mitigating harmful mistranscriptions toward unwanted token sequences. Similar parameter is implemented in the HuggingFace package: https://huggingface.co/docs/transformers/en/internal/generation_utils#transformers.SequenceBiasLogitsProcessor

### CODE REVIEW

**Comment by MrigankRaman** [comment]
When will this be released?
